src/Interfaces/MarketingSuite/HasSource.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types = 1);
+
+namespace Vicimus\Support\Interfaces\MarketingSuite;
+
+use Assets\Models\Source;
+use Illuminate\Database\Eloquent\Relations\MorphMany;
+use Illuminate\Database\Eloquent\Relations\MorphOne;
+
+/**
+ * Interface HasSource
+ *
+ *
+ */
+interface HasSource
+{
+    /**
+     * Has many assets
+     * @return MorphMany|AssetModel[]
+     */
+    public function assets(): MorphMany;
+
+    /**
+     * Has many audiences
+     * @return MorphMany
+     */
+    public function audiences(): MorphMany;
```

Comment:
This is wierd. I thought the source itself has audiences and campaigns dont have any audiences.